special-interest-groups/observability-tag/charter.md

```diff
@@ -0,0 +1,48 @@
+# OpenSearch Observability Technical Advisory Group (TAG) Governance Charter
+
+### Purpose
+
+The OpenSearch Project proposes to create a Observability Interest Group that focuses on observability using OpenSearch. This includes log analytics, traces, metrics using OpenSearch and related projects such as Prometheus, OpenTelemetry. This group will also establish guidelines for members, set architecture guidelines, and come up with recommendations which will then be reviewed by TSC for decision making. The group will provide feedback to the TSC before the final decision is made through voting of TSC members.
+
+### Responsibilities
+
+The following is a non-exhaustive sample list of activities and deliverables that are in-scope for this SIG
+
+* Drive feature development and enhancements for observability applications using OpenSearch (e.g. ingestion, plugins, Alerting, Anomaly Detection, Dashboards).
+* Align OpenSearch observability initiatives with industry trends, CNCF observability guidelines, and interoperability standards.
+* Collaborate with external observability projects like OpenTelemetry, Prometheus, Jaeger, and Fluent Bit to enable seamless data ingestion, integration and interoperability. 
+* Define and promote observability best practices for OpenSearch deployments, covering data ingestion pipelines, alerting, visualization, and operational monitoring.
```

Comment:
This adds another layer of responsibility.  In the _Purpose_ section, only using OpenSearch as a tool for observability is mentioned. Here the scope is extended to observing OpenSearch as well. To me this is fine for now, since it is very likely, that the same knowledge and experts would be working on those topics. But this responsibility might be split from the TAG/SIG now or at a later stage.